app/controllers/api/v1/audience_segments_controller.rb

```diff
@@ -0,0 +1,90 @@
+module Api
+  module V1
+    class AudienceSegmentsController < ApiController
+      DEFAULT_PER_PAGE = 30
+      MAX_USER_IDS = 10_000
+      USER_ATTRIBUTES_FOR_SERIALIZATION = %i[
```

Comment:
A bit of an aside but it would be nice to have this user serialisation logic encapsulated somewhere - every controller (now including this one) that needs to serialise users has been duplicating it.

Same with the `API_PER_PAGE_MAX` pagination constraint.